src/main/java/org/fogbowcloud/arrebol/core/models/specification/Specification.java

```diff
@@ -208,8 +210,7 @@ public boolean equals(Object obj) {
 
     public Specification clone() {
         Specification cloneSpec = new Specification(this.image, this.username, this.publicKey, this.privateKeyFilePath,
-                this.cloudName, this.userDataFile, this.userDataType);
-        cloneSpec.putAllRequirements(this.getAllRequirements());
+                this.requirements, this.cloudName);
```

Comment:
why do ne need this method?

Reply:
This method was removed. It was unnecessary.